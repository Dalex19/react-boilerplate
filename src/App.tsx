import React from 'react'
import AppView from './app-view'

function App() {
  return (
      <AppView />
  );
}

export default App
