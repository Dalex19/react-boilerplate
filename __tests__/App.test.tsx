import React from "react";
import { render } from "@testing-library/react";
import App from "../src/App";

describe('<App />', function () {
    test('should render the App component', function () {
        const { asFragment } = render(<App />);

        expect(asFragment()).toMatchSnapshot();
    });
});
