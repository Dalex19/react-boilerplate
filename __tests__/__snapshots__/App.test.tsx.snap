// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render the App component 1`] = `
<DocumentFragment>
  <div
    class="app"
  >
    <div
      class="font-display flex h-screen bg-gray-50"
    >
      <div
        class="min-h-screen mx-auto"
      >
        <form
          class="card align-middle rounded-lg mt-52 container md:p-24 bg-white"
        >
          <div
            class="mx-auto"
          >
            <p
              class="pb-6 -mt-2 text-xl text-center"
            >
              Sign In
            </p>
          </div>
          <div
            class="login-form"
          >
            <input
              class="shadow-sm appearance-none focus:border-white border border-gray-100 rounded w-full py-2 mt-2 input"
              name="username"
              placeholder="username"
              type="text"
            />
            <input
              class="shadow-sm appearance-none focus:border-white border border-gray-100 rounded w-full py-2 mt-2 input"
              name="password"
              placeholder="password"
              type="password"
            />
            <input
              class="focus: shadow-sm w-full mt-6 btn bg-green-600 rounded text-white p-2"
              type="submit"
              value="Sign in"
            />
          </div>
        </form>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
